c2corg_api/views/user.py

```diff
@@ -442,7 +461,7 @@ def __init__(self, request):
         validators=[colander_body_validator, validate_json_password])
     def post(self):
         request = self.request
-        username = request.validated['username']
+        username = request.validated['username'].strip().lower()
```

Comment:
Not needed if eventually we decide not to convert the existing usernames with leading or trailing whitespaces.